refaster-support/src/main/java/tech/picnic/errorprone/refaster/matchers/IsRefasterAsVarargs.java

```diff
@@ -0,0 +1,23 @@
+package tech.picnic.errorprone.refaster.matchers;
+
+import static com.google.errorprone.matchers.Matchers.staticMethod;
+
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.matchers.Matcher;
+import com.google.errorprone.refaster.Refaster;
+import com.sun.source.tree.ExpressionTree;
+
+/** A matcher of {@link Refaster#asVarargs} method invocations. */
+public final class IsRefasterAsVarargs implements Matcher<ExpressionTree> {
+  private static final long serialVersionUID = 1L;
+  private static final Matcher<ExpressionTree> DELEGATE =
+      staticMethod().onClass(Refaster.class.getName()).namedAnyOf("asVarargs");
```

Comment:
When passing only one, we can use `#named` 😄.
```suggestion
      staticMethod().onClass(Refaster.class.getName()).named("asVarargs");
```